lib/Steps/Integrations/Cordova.ts

```diff
@@ -190,31 +190,68 @@ export class Cordova extends BaseIntegration {
         shellPath: '/bin/sh',
         shellScript:
           // eslint-disable-next-line prefer-template
+          '#!/bin/bash\\n' +
+          '# print commands before executing them and stop on first error\\n' +
+          'set -x -e\\n' +
           'echo "warning: uploading debug symbols - set SENTRY_SKIP_DSYM_UPLOAD=true to skip this"\\n' +
-          'if [ -n "$SENTRY_SKIP_DSYM_UPLOAD" ]; then\\n' +
-          '  echo "warning: skipping debug symbol upload"\\n' +
-          '  exit 0\\n' +
-          'fi\\n' +
-          'export SENTRY_PROPERTIES=' +
-          cwd +
-          '\\n' +
-          'function getProperty {\\n' +
-          '    PROP_KEY=$1\\n' +
-          '    PROP_VALUE=`cat $SENTRY_PROPERTIES | grep "$PROP_KEY" | cut -d\'=\' -f2`\\n' +
-          '    if [ -z "$PROP_VALUE" ]; then\\n' +
-          '        echo "plugins/sentry-cordova/node_modules/@sentry/cli/bin/sentry-cli"\\n' +
-          '    else\\n' +
-          '        echo $PROP_VALUE\\n' +
+          '[ -z "$SENTRY_FORCE_FOREGROUND"] && SENTRY_FORCE_FOREGROUND=true\\n' +
+          '[[ "$SENTRY_FORCE_FOREGROUND" == true ]] && SENTRY_FORCE_FOREGROUND_FLAG="--force-foreground"\\n' +
```

Comment:
Since https://github.com/getsentry/sentry-cli/releases/tag/2.37.0 the upload is always in foreground.